.rubocop.yml

```diff
@@ -1,14 +1,52 @@
+require: rubocop-jekyll
 inherit_gem:
-  jekyll: .rubocop.yml
+  rubocop-jekyll: .rubocop.yml
+
+AllCops:
+  TargetRubyVersion: 2.3
+  Exclude:
+    - docs/**/*
+    - script/**/*
+    - test/**/*
```

Comment:
Why do we not want rubocop to run on tests?

Reply:
OK, will deal with that part in a dedicated PR 😄